src/graphql/context.cr

```diff
@@ -2,14 +2,16 @@ require "./language"
 
 module GraphQL
   class Context
-    property max_complexity : Int32?
+    property max_complexity : Int32? = nil
     property complexity = 0
     property fragments : Array(Language::FragmentDefinition) = [] of Language::FragmentDefinition
     property query_type : String = ""
     property mutation_type : String? = nil
     property document : Language::Document?
 
-    def initialize(@max_complexity = nil)
+    # Return string message to be added to errors object or throw to bubble up
+    def handle_exception(ex : Exception) : String?
+      ex.message
     end
```

Comment:
oh, clever... So we would just override this in our own context, then we could propagate whichever error we want? I like that.